packages/terra-clinical-onset-picker/tests/wdio/onset-spec.js

```diff
@@ -1,5 +1,6 @@
 const viewports = Terra.viewports('tiny', 'small', 'medium', 'large', 'huge', 'enormous');
```

Comment:
Is there a reason for every single viewport? This component only responsively changes in tiny viewports, yet 288 tests run. To cover the same area, we could cut this to tiny, medium, enormous

Reply:
Changed viewports to tiny, medium, and enormous.